generators/app/index.js

```diff
@@ -1,14 +1,16 @@
-import _ from 'lodash';
-import chalk from 'chalk';
-import path from 'path';
-import tsmlj from 'tsmlj';
-import yeoman from 'yeoman-generator';
-import yosay from 'yosay';
-import shelljs from 'shelljs';
-
-import {PREFIX} from './constants';
-import packageJSON from './package-json';
-import * as validators from './validators';
+'use strict';
```

Comment:
I think all node modules have this on by default. So I don't think we need to specify this here or in the other files.